psi4/driver/diatomic.py

```diff
@@ -95,6 +95,11 @@ def anharmonicity(rvals, energies, plot_fit='', mol = None):
         raise ValidationError("At least 5 data points must be provided to compute anharmonicity")
     core.print_out("\n\nPerforming a fit to %d data points\n" % npoints)
 
+    # Sort radii and values first from lowest to highest radius
+    sortedtuples = sorted(zip(rvals,energies))
```

Comment:
```
idx = np.argsort(rvals)
rvals = rvals[idx]
energies = energies[idx]
```

Reply:
I had to wrap the python arrays around `np.array()`, hope that's ok.